### ISSUE TITLE
Deaf trading

### ISSUE THREAD

**Issue by PGR-14** [CleverRaven/Cataclysm-DDA#75273]
### Is your feature request related to a problem? Please describe.

Played a deaf character recently, but had no way to trade, would probably make sense for a deaf person to be able to trade through hand movements & gestures

### Solution you would like.

Add it in, reusing code from the Churl profession/mutation/scenario

**Comment by Terrorforge** [comment]
As of #74165 there's also a system for relearning human speech after acquiring inhuman anatomy which could probably be applied to sign language, including a basic "point at thing and hold up fingers" version

**Comment by PGR-14** [comment]
I don't think we really need a proficiency for it, that's basic communication. But you could make the check based on A: Your proficiency B: Your intelligence & C: The target's intelligence.
